File: android/BettingHUB/app/src/main/java/com/xbethub/webview/models/Forecast.kt
package com.xbethub.webview.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable


data class Forecast (
    @SerializedName("id")
    @Expose
    val id: Int,

    @SerializedName("user_data")
    @Expose
    val user: User,

    @SerializedName("event_data")
    @Expose
    val event: Event,

    @SerializedName("forecast_text")
    @Expose
    val text: String,

    @SerializedName("forecast_created_at")
    @Expose
    val createdAt: String,

    @SerializedName("bet_data")
    @Expose
    val bet: Bet,

    @SerializedName("forecast_stats")
    @Expose
    val stats: ForecastStats

): Serializable {}

File: src/BettingParser/Models/Forecast.cs
using System;
using Newtonsoft.Json;

namespace BettingParser.Models
{
    [Serializable]
    public class Forecast
    {
        [JsonProperty("parser_forecast_id")]
        public string ParserForecastId { get; set; }
        
        [JsonProperty("event_data")]
        public EventData EventData { get; set; }
        
        [JsonProperty("forecast_text")]
        public string ForecastText { get; set; }
        
        [JsonProperty("bet_data")]
        public BetData BetData { get; set; }

        [JsonProperty("bookmaker_name")]
        public string BookmakerName { get; set; }
    }
}